clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleLoadBalancerProvider.groovy

```diff
@@ -259,7 +270,11 @@ class GoogleLoadBalancerProvider implements LoadBalancerProvider<GoogleLoadBalan
         loadBalancerPort = portString
         break
       case GoogleLoadBalancerType.SSL:
-        instancePort = 'http' // NOTE: This is what occurs in Google Cloud Console, it's not documented and a bit non-sensical.
```

Comment:
I believe this is just for display purpose in deck. For SSL/TCP proxies, both instance port and load balancer port must be the same.